.github/workflows/config/spelling_allowlist.txt

```diff
@@ -86,6 +89,7 @@ SLED
 SLES
 SLURM
 SVD
+Superpositions
```

Comment:
To facilitate finding words in this list, our pipelines expect the entries in this file to be alphabetically sorted.